vlei-verifier-workflows/src/utils/test-util.ts

```diff
@@ -627,83 +629,29 @@ export async function sendAdmitMessage(
   await markAndRemoveNotification(senderClient, grantNotification);
 }
 
-export async function getRootOfTrust(configJson: any): Promise<any> {
-  if (hasGLEIFWithMultisig(configJson)) {
-    return await getRootOfTrustMultisig(configJson);
-  } else {
-    return await getRootOfTrustSinglesig(configJson);
-  }
-}
-
-function hasGLEIFWithMultisig(data: any): boolean {
-  return data.users.some(
-    (user: any) =>
-      (user.type === "GLEIF" || user.type === "GLEIF_EXTERNAL") &&
-      user.identifiers.some((id: any) => data.identifiers[id]?.identifiers),
-  );
-}
+export async function getRootOfTrust(configJson: any, rot_aid: string, rot_member_aid?: string): Promise<any> {
+  const workflow_state = WorkflowState.getInstance(configJson);
+  const rotAID = workflow_state.aids.get(rot_aid);
+  let client;
 
-async function getRootOfTrustMultisig(configJson: any): Promise<any> {
-  const rootOfTrustMultisigIdentifierName = configJson.users
-    .filter(
-      (usr: any) => usr.type == "GLEIF" || usr.type == "GLEIF_EXTERNAL",
-    )[0]
-    .identifiers.filter((identifier: string) =>
-      identifier.includes("multisig"),
-    )![0];
-
-  const rootOfTrustIdentifierName = configJson.users
-    .filter(
-      (usr: any) => usr.type == "GLEIF" || usr.type == "GLEIF_EXTERNAL",
-    )[0]
-    .identifiers.filter(
-      (identifier: string) => !identifier.includes("multisig"),
-    )![0];
-
-  const rootOfTrustIdentifierAgent =
-    configJson.agents[configJson.identifiers[rootOfTrustIdentifierName].agent];
-  const rootOfTrustIdentifierSecret =
-    configJson.secrets[rootOfTrustIdentifierAgent.secret];
-  const clients = await getOrCreateClients(
-    1,
-    [rootOfTrustIdentifierSecret],
-    true,
-  );
-  const client = clients[clients.length - 1];
-  const rootOfTrustAid = await client
-    .identifiers()
-    .get(rootOfTrustMultisigIdentifierName);
+  const rootOfTrustIdentifierName = configJson.users.find(
```

Comment:
What do you think about the root of trust being part of WorkflowState so that we don't have to find by name? Meaning that any AID could be designated as the root of trust. Perhaps you and @aydarng discussed this already. I'm open to ideas.